ext/intl/converter/converter.stub.php

```diff
@@ -11,7 +11,7 @@ public function convert(string $str, bool $reverse = false) {}
 
     /**
      * @param int $error
-     * @return string|null
+     * @return string|int|array|null
```

Comment:
Why? I can only see string or null as possible return types.

Reply:
Yeah, I also had this impression, but fortunately there's an RFC for this which has some documentation for the return types: https://wiki.php.net/rfc/uconverter#advanced_use

Reply:
@sgolemon Maybe you would be able to add to the conversation here as RFC author. What are the valid return types for this method?

Reply:
The RFC states that:

> Return values for this method may be: NULL, Long, String, or Array. A value of NULL indicates that the codepoint/codeunit should be ignored and left out of the destination/intermediate string. A Long return value will be treated as either a Unicode codepoint for toUCallback(), or a single-byte character in the target encoding for fromUCallback(). A String return value will be treated as one (or more) UTF8 encoded codepoints for toUCallback(), or a multi-byte character (or characters) in the target encoding for fromUCallback().

Children can override these methods, so I guess only the parent's return value is restricted to a string.

Reply:
See also https://github.com/php/php-src/blob/4baf340c3984239da20866bdc122c0b0df84c0f4/ext/intl/converter/converter.c#L254, where this logic is implemented.